plan/config.go

```diff
@@ -104,10 +120,35 @@ func parseBehavior(d string) Behavior {
 		ClientAxis: clientAxis,
 		ParamsAxes: values,
 	}
-
 	return behavior
 }
 
+func parseSkipBehavior(d string) (string, []Filter, error) {
+	pair := strings.SplitN(d, "=", 2)
+	if len(pair) != 2 {
+		return "", nil, fmt.Errorf("Invalid input: %s", d)
```

Comment:
Also, when referencing an input string in error messages, it's often useful to use `%q` rather than `%s` because it will quote the string so you'll have a reasonable error message even if the string is empty.

    invalid input ""